docs/src/main/tut/monoidk.md

```diff
@@ -25,3 +25,37 @@ Here's how to distinguish `Monoid` and `MonoidK`:
     also means that for any `A`, there is an "empty" `F[A]` value. The
     combination operation and empty value just depend on the
     structure of `F`, but not on the structure of `A`.
+
+Let's compare the usage of `Monoid[A]` and `MonoidK[F]`.
+
+First some imports:
+
+```tut:silent
+import cats.{Monoid, MonoidK}
+import cats.instances.list._
```

Comment:
In #1026 we settled on using the "uber import" (`import cats.implicits._`) in the documentation. For the rest :+1: